### DISCUSSION TITLE
GraphScope中的图分析引擎GAE后续规划

### DISCUSSION THREAD

**Discussion by dakeleblack** [alibaba/GraphScope#3914] [Q&A] [answered]
GraphScope中的图分析引擎GAE后续规划是什么样的，在计算性能方面是否会再进一步优化

**Comment by yecol** [comment, marked as answer]
hi @dakeleblack 
感谢你对 GraphScope 的使用和关注！～ 
后续我们会借助 mpi-operator 提供 Job 的方式直接跑 analytics 的任务。

从各种测试数据上看，计算性能上我们的引擎是保持领先优势的。
请问你是在使用过程中在计算性能方面遇到瓶颈吗？

**Reply by dakeleblack** [reply]
看到目前的计算大多数均在内存中进行，不知道对于内存+外存的计算模式是否会有进一步的研究

**Reply by JackyYangPassion** [reply]
补充一点：如果是全图 比如 1TB 点边，预示着占用内存要5TB
